processing.py

```diff
@@ -0,0 +1,83 @@
+#!/bin/python3
+import os
+import json
+
+processing_dir = "packages/schema/src/schema/processing"
+
+filenames = [file for file in os.listdir(processing_dir) if file.endswith(".json")]
+
+params = []
+for filename in filenames:
+    filename = f"{processing_dir}/{filename}"
+    with open(filename, "r") as f:
+        e = json.loads(f.read())
+        keep = [
+            "description",
+            "gen_name",
+            "gen_params",
+            "gen_label",
+            "gen_type",
+            "gen_additionals",
+        ]
+        params.append({x: e[x] for x in keep})
+
+
+def addBanner(f):
+    f.write("//Generated automatically please don't modify directly\n")
+
+
+def generateJsonMerge():
+    output = f"packages/schema/src/processing/_generated/processing_merge.json"
+
+    with open(output, "w+") as f:
+        json.dump(params, f, indent=2)
+        f.write("\n")
+
+
+def exportSchema():
+    curFileName = "packages/schema/src/processing/_generated/exportProcessingSchema.ts"
+    with open(curFileName, "w+") as f:
+        addBanner(f)
+        for param in params:
+            gen_name = param["gen_name"]
+            f.write(f"export * from '../../_interface/processing/{gen_name}';\n")
```

Comment:
We may need to keep these in a Python script. But since it's only touching the schema folder, we should move the script under `packages/schema/scripts` and run it as part of the `build` script of the schema package. That way the package is properly self-contained.